modules/react-components/src/components/input/url-input.tsx

```diff
@@ -414,7 +414,7 @@ export const URLInput: FunctionComponent<URLInputPropsInterface> = (
                 popupSubHeader={
                     <React.Fragment>
                         <Icon name={ positive ? "check" : "times" } color={ positive ? "green" : "red" }/>
-                        { origin }
+                        { origin !== "null" ? origin : href }
```

Comment:
Addressed with 86eb928142877ddb0c7c68ea35ed93a18ea55395.
The null checker is added to avoid displaying `null` in the popup window specially for mobile urls.